research-software-directory/RSD-as-a-service
# 453 Research domains to show key and title

# Research domains

Research domains based on ERC classification has very long text definitions. On ERC website they use in most cases short names (keys). We decided to use keys and show part of the description as much as possible. The complete description is shown on mouse-over.

Closes #453 

Changes proposed in this pull request:
*  Use research domain keys and title    

How to test:
* `make start` to rebuild everything
*  open rsd on localhost
*  confirm that project page shows research domains as key and title.

## Example research domains

![image](https://user-images.githubusercontent.com/9204081/195896152-d371fc47-27b4-4b45-af24-d7bbb61fe4e6.png)

PR Checklist:

*   [] Increase version numbers in `docker-compose.yml`
